client/rest-high-level/src/main/java/org/elasticsearch/client/MachineLearningClient.java

```diff
@@ -247,4 +249,40 @@ public void closeJobAsync(CloseJobRequest request, RequestOptions options, Actio
             listener,
             Collections.emptySet());
     }
+
+    /**
+     * Gets the buckets for a Machine Learning Job.
+     * <p>
+     * For additional info
+     * see <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-get-bucket.html">ML GET buckets documentation</a>
+     *
+     * @param request  the request
```

Comment:
nit: capitalise 'The'.  Same in the comment for `getBucketsAsync`

Reply:
Actually reviewing further down the lower case start of the parameter description is more consistent. If that is the preferred style change 'Additional' on the line below and in the async version comments

Reply:
it's just vastly inconsistent :-(

Reply:
I like capitalized as it separates better from the param name. I'll capitalize and cross my fingers.